nssp/tests/test_data/page.json

```diff
@@ -196,5 +196,41 @@
         "fips": "8101",
         "trend_source": "HSA",
         "buildnumber": "2025-02-28"
+    },
+    {
+        "_comment":"This record is for testing the case where all signals data is NA for a county",
```

Comment:
You are leaving out some fields here to make it appear like the values are missing...  Should we include (at least some of) the fields but with empty string values to make it look more like what we get from data.cdc.gov?